.github/workflows/check.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 The NMPolicy Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+name: checks
+on: [push, pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - uses: arnested/go-version-action@v1
+        id: go-version
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ steps.go-version.outputs.minimal }}
+      - name: Run linter
+        run: make lint
```

Comment:
I think it would be better to run it from prow, the same way done with e2e-tests and  unit-tests
What do you think?